ownCloud/Client/Viewer/Media/MediaDisplayViewController.swift

```diff
@@ -32,7 +32,7 @@ class MediaDisplayViewController : DisplayViewController {
 	private var playerStatusObservation: NSKeyValueObservation?
 	private var playerItemStatusObservation: NSKeyValueObservation?
 	private var playerItem: AVPlayerItem?
-	private var player: AVPlayer?
+	var player: AVPlayer?
```

Comment:
@hosy eventually don't expose the player but may be add some properties you need accessing `AVPlayer`? As an example I've seen using `AVPlayer.rate` for checking if it is playing something? Instead you could add `MediaDisplayViewController.isPlaying`

Reply:
Fixed!